src/PrestaShopBundle/Resources/views/Admin/Sell/Attachment/edit.html.twig

```diff
@@ -0,0 +1,44 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% set enableSidebar = true %}
+
+{% set layoutTitle = 'Editing attachment %name%'|trans({'%name%': attachmentInformation.name|first}, 'Admin.Orderscustomers.Feature') %}
```

Comment:
Where would this wording be displayed? It is not the mere "Edit" button at the right end of each line in the files list, right?

Reply:
It is show as title in edit attachment page and now is changed to Edit: %value% from Admin.Catalog.Feature domain